arch/riscv/core/smp.c

```diff
@@ -84,7 +84,11 @@ void arch_secondary_cpu_init(int hartid)
 
 #ifdef CONFIG_SMP
 
-#define MSIP_BASE 0x2000000UL
+#define CLINT_NODE DT_NODELABEL(clint)
+#if !DT_NODE_EXISTS(CLINT_NODE)
+#error "clint label is not defined in the dts."
```

Comment:
I'd probably just use the term 'devicetree'

```suggestion
#error "clint label is not defined in the devicetree."
```

Reply:
OK.

Reply:
If I can use rebase and 'push -f' to accept the fix?

Reply:
If I can use 'commit --amend' and 'push -f' to accept the fix?

Reply:
What I'd do:
1. `git rebase -i HEAD~2`
2. leave the first commit as `pick`
3. `squash` the second commit
4. keep the commit title/msg/sign-off of the first commit
5. force push

Reply:
AI tells me PR is matching my forked branch. Know more. good.

Reply:
Thanks.